### PR TITLE
Add masterclass automated ad template

### PR DESCRIPTION
**Pull request by buck06191**
This does the following:
 * Make the `events-multiple.js` file tone agnostic. Endpoints are set
   using the tone of the event type.
 * Add a `masterclass` ad type - this uses the correct tone.
 * Small formatting changes.
 * Updated `README.md` to mention locally testing new ad format.

Requires https://github.com/guardian/membership-frontend/pull/1918 to have been merged in order for this to function correctly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ This will require you to use:
 - the [`portify`](https://github.com/guardian/commercial-templates/blob/master/src/_shared/js/dev.js) method in `src/_shared/js/dev.js` to programmatically switch between 7000 and 9000 when making outbound requests.
 - the [`JsonComponent`](https://github.com/guardian/frontend/blob/master/common/app/common/JsonComponent.scala) method to serve the JSON response from the frontend controller. This method wraps the JSON response in a [CORS](https://github.com/guardian/frontend/blob/master/common/app/model/Cors.scala) header which allows cross-origin requests.
 
+### Automated ads
+
+A number of ads (`events-multiple` and `masterclass-multiple`) use areader revenue endpoint. In their associated `test.json` files you will see that the URL's refer to `mem.thegulocal.com` as the domain. This requires the [`membership-frontend`](https://github.com/guardian/membership-frontend) to be running and using the `nginx` server. Make sure to set this up before testing these ads.
```
Comment (ioanna0) [inline README.md]: tiny typo : 'a reader'

File: src/_shared/js/events-multiple.js [hunk 2]
```diff
@@ -59,18 +62,18 @@ function importCard(adType) {
   }
 }
 
-const boldTitle = (title) => {
-    return title.split(":")[1] ? `<b>${title.split(":")[0]}:</b>${title.split(":")[1]}` : `<b>${title}</b>`
-}
+const boldTitle = title => {
+  return title.split(":")[1]
```
Comment (ioanna0) [inline src/_shared/js/events-multiple.js]: what happens if theres no `title` or a `title` without a semicolon?

File: src/masterclass-multiple/web/index.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+@import '_adverts-networks';
+@import '_adverts-shop';
+@import '_adverts-money';
+@import '_advert';
+@import '_adverts-futurelearn';
+@import '_adverts-weekly';
+@import '_adverts-members';
+@import '_adverts-patron';
+@import '_adverts-lifestyle';
+@import '_adverts-climate';
```
Comment (faresite) [inline src/masterclass-multiple/web/index.scss]: Do you need all this CSS for the masterclass slots?

File: src/_shared/js/events-multiple.js (hunk 2 above)
Comment (buck06191) [reply src/_shared/js/events-multiple.js]: Title should always exist. The Scala endpoint has it as `String` rather than `Option[String]`

**Pull request merged**